Translator/Translator/Source File/triangle.cpp

```diff
@@ -11,3 +11,7 @@ void Triangle::get_coord() const {
     p2.get_coord();
     p3.get_coord();

```

Comment:
Same as point 
Triangle is a class that creates a triangle and should not have a print functionality.

Reply:
I understand your concern, but I think the print functionality within the Triangle class serves our current needs. It's convenient for quickly displaying the triangle's coordinates, and aligns with how we're managing output across other parts of the project